examples/jsm/math/Lut.js

```diff
@@ -75,8 +76,8 @@ class Lut {
 					const min = this.map[ j ][ 0 ];
 					const max = this.map[ j + 1 ][ 0 ];
 
-					minColor.set( this.map[ j ][ 1 ] );
-					maxColor.set( this.map[ j + 1 ][ 1 ] );
+					minColor.setHex( this.map[ j ][ 1 ], LinearSRGBColorSpace );
+					maxColor.setHex( this.map[ j + 1 ][ 1 ], LinearSRGBColorSpace );
```

Comment:
The lerp should be done with sRGB colors so I use this approach to avoid a color space conversion when setting the hex values. I'm open for different approach since using `LinearSRGBColorSpace` is a bit confusing at first glance.